vignettes/basics.Rmd

```diff
@@ -0,0 +1,107 @@
+---
+title: "Basics of shiny.semantic"
+author: "Appsilon"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Basics of shiny.semantic}
+  %\VignetteEngine{knitr::knitr}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE, screenshot.force = FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  comment = "#>"
+)
+library(shiny)
+library(shiny.semantic)
+```
+
+
+In this tutorial we will walk you through the basics of `shiny.semantic`. 
+
+## Quick Q&A
+
+First of all, let's answer a few questions:
+
+- **Why do I need `shiny.semantic`?**
+
+You can build your `shiny` apps with standard shiny UI that relies on [Bootstrap](https://getbootstrap.com/).
```

Comment:
General comment -> I'm not sure if shiny should be lower case "s" or "Shiny".

Reply:
Yes, probably should be capital as in RStudio docs